src/pages/submit/linked-wearables.tsx

```diff
@@ -414,7 +530,7 @@ export default function SubmitLinkedWearables() {
           disabled={formDisabled}
         />
       </ContentSection>
-      {getListSection({ section: 'managers', type: 'address' })}
+      {getListSection({ section: 'managers', type: 'address' }, addressValidator)}
```

Comment:
I'm wondering if we should have a MAX amount for all the other fields like links, smart contract and managers